spacy/tests/lang/en/test_exceptions.py

```diff
@@ -1,4 +1,159 @@
 import pytest
+from spacy.matcher import Matcher
+from spacy.tokens import Span
+
+
+@pytest.mark.issue(351)
+def test_issue351(en_tokenizer):
```

Comment:
I've put some of the tokenizer regressions in the English language tests instead of `tests/tokenizer/test_tokenizer.py`